src/components/CommentDelete/CommentDelete.jsx

```diff
@@ -0,0 +1,23 @@
+import { deleteComment } from "../../api";
+
+const CommentDelete = ({ comment, setComments }) => {
+
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    deleteComment(comment.comment_id)
+    
+    setComments((currComment) => currComment.filter((e)=> {
+      return e.comment_id != comment.comment_id
+    }))
```

Comment:
When we click delete, to avoid any errors, butter to setComments in the .then() block of deleteComment to make sure it's updated after the call is successfully made.